aspnet/mvc/controllers/testing/sample/TestingControllersSample/tests/TestingControllerSample.Tests/IntegrationTests/SessionControllerIndex.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System.IO;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.AspNet.TestHost;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.PlatformAbstractions;
+using TestingControllersSample;
+using Xunit;
+
+namespace TestingControllerSample.Tests.IntegrationTests
+{
+    public class SessionControllerIndex
```

Comment:
Yes, we want to be consistent with the engineering team's guidance. We're not trying to mandate that everyone use one particular style, but within our repos we want to be consistent. 

The reason the engineering team prefers the one file approach is that the controller setup logic will typically be the same or very similar across all actions in a controller, so it's easier to share this logic if it all lives in the same file. For example, both the `HomeControllerIndex` and `HomeControllerIndexPost` files have copies of the `GetTestSessions` method, which seems redundant.